code/game/objects/effects/landmarks.dm

```diff
@@ -588,6 +588,62 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/awaystart) //Without this away mission
 	ruin_template = null
 	. = ..()
 
+
+/// Mob spawners, spawns a mob and deletes the landmark
+
+/obj/effect/landmark/mob_spawner
+	///The mob we use for the spawner
+	var/mobtype = null
+
+/obj/effect/landmark/mob_spawner/Initialize(mapload)
+	. = ..()
```

Comment:
I don't think these even need to call the parent since they're just going to be immediately deleted.

Reply:
Applied

Reply:
linters doesn't play nice with this, even if it does pretty much nothing